germany_fidorbank/README.rst

```diff
@@ -0,0 +1,91 @@
+~~~~~~~~~~~~~~~~~~~~~~~
+ofxstatement-fidorbank UPDATE THIS README
```

Comment:
Can you please remove/update this "UPDATE THIS README"

Reply:
It's already been updated so this is safe to remove